python/ray/autoscaler/aws/example-gpu-docker.yaml

```
@@ -1,12 +1,8 @@
 # An unique identifier for the head node and workers of this cluster.
-cluster_name: gpu-docker
-
-# The minimum number of workers nodes to launch in addition to the head
-# node. This number should be >= 0.
-min_workers: 0
+cluster_name: gpu-dockeraaaa
```

Is this intentional? 
```suggestion
cluster_name: gpu-docker
```

---

thanks for catching that -- I changed the name while testing to simultaneously test two version of this, forgot to change it  back

---

fixed